hikari/impl/bot.py

```diff
@@ -844,6 +864,15 @@ async def update_presence(
 
         await aio.all_of(*coros)
 
+    async def _set_close_flag(self, signame: str, signum: int) -> None:
```

Comment:
if this is becoming a method, perhaps `async def handle` above should be too?

Reply:
I moved this here to make `run` shorter as it was too populated. I also tried to move `handle`, but since it accesses variables that are on `run`, I decided not to. I can move it if you want